src/modules/endpoint/DatabaseConnector.js

```diff
@@ -187,6 +197,40 @@ class DatabaseConnector {
     await this.writeFile(path, JSON.stringify(languages))
   }
 
+  async storeCities (cities: Array<CityModel>) {
+    const jsonModels = cities.map((city: CityModel): ContentCityJsonType => ({
+      name: city.name,
+      live: city.live,
+      code: city.code,
+      extrasEnabled: city.extrasEnabled,
+      eventsEnabled: city.eventsEnabled,
+      sortingName: city.sortingName
+    }))
+
+    await this.writeFile(this.getMetaCitiesPath(), JSON.stringify(jsonModels))
+  }
+
+  async loadCities (): Promise<Array<CityModel> | null> {
+    const path = this.getMetaCitiesPath()
+    const fileExists: boolean = await RNFetchblob.fs.exists(path)
+
+    if (!fileExists) {
+      return null
+    }
+
+    const cities = JSON.parse(await this.readFile(path))
+    return cities.map(city =>
```

Comment:
There might also be the case that the `lastUpdate` was already stored for the city, but the rest of the city meta data is missing. So make sure that nothing is `undefined` here.